src/components/landing/Features.tsx

```
@@ -66,51 +58,86 @@
-                          __html: feature.title,
-                        }}
-                      />
-                    </div>
+                <div className="flex flex-col h-full w-full">
+                  <div className="mb-5 ">
+                    <feature.icon className="w-12 h-12 hover:scale-110 transition-all duration-100" />
+                  </div>
+                  <div className="flex-grow ">
+                    <p
+                      className="max-w-lg text-xl font-normal tracking-tighter md:text-2xl"
+                      dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)